front/app/translations/admin/ro-RO.json

```diff
@@ -1,10 +1,15 @@
 {
   "UI.FormComponents.required": "obligatoriu",
+  "app.components.Admin.ImageCropper.imageSupportPageText": "visit our support center",
+  "app.components.Admin.ImageCropper.imageSupportPageURL": "https://support.citizenlab.co/en/articles/1346397-what-are-the-recommended-dimensions-and-sizes-of-the-platform-images",
+  "app.components.Admin.ImageCropper.imageTooltip": "For more information on recommended image resolutions, {supportPageLink}.",
+  "app.components.Admin.ImageCropper.info": "This image is always cropped to a certain ratio to make sure all crucial aspects are on display at all times. The {link} for this image type is {aspect}:1.",
+  "app.components.Admin.ImageCropper.infoLinkText": "recommended ratio",
```

Comment:
This file does not seem to exist on Crowdin master. See: https://crowdin.com/project/citizenlabdotcocitizenlab/ro

Somehow this has been included (probably due to my mistaken merging of crowdin branch PR to my branch before merging my branch to master). Should we remove it? How? By creating a PR to do this? (and for any other such files)